bazel/rules_python.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/python/pip_install/repositories.bzl b/python/pip_install/repositories.bzl
+index df63674..80824e4 100644
+--- a/python/pip_install/repositories.bzl
++++ b/python/pip_install/repositories.bzl
+@@ -16,8 +16,8 @@ _RULE_DEPS = [
+     ),
+     (
+         "pypi__setuptools",
+-        "https://files.pythonhosted.org/packages/54/28/c45d8b54c1339f9644b87663945e54a8503cfef59cf0f65b3ff5dd17cf64/setuptools-42.0.2-py2.py3-none-any.whl",
+-        "c8abd0f3574bc23afd2f6fd2c415ba7d9e097c8a99b845473b0d957ba1e2dac6",
++        "https://files.pythonhosted.org/packages/3d/f2/1489d3b6c72d68bf79cd0fba6b6c7497df4ebf7d40970e2d7eceb8d0ea9c/setuptools-51.0.0-py3-none-any.whl",
```

Comment:
perhaps we should bump this to `54.0` as that is current stable i think

Reply:
Can you fix this upstream? We really dislike patches (see https://github.com/envoyproxy/envoy/blob/main/DEPENDENCY_POLICY.md#dependency-patches)

Reply:
ive raised a ticket already:

https://github.com/bazelbuild/rules_python/issues/421

ill experiment with using rules_python from git - not sure if that will fix the problem tho - so this patch may be necessary for now